sampling/hierarchical/test/opentelemetry/sampling/hierarchical/sampler_test.rb

```diff
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require('test_helper')
+
+describe(OpenTelemetry::Sampling::Hierarchical::Sampler) do
+  describe('#should_sample?') do
+    it('returns DROP if it has no samplers') do
+      sampler = OpenTelemetry::Sampling::Hierarchical::Sampler.new([])
+
+      _(
+        sampler.should_sample?(
+          trace_id: SecureRandom.uuid.to_s,
```

Comment:
Consider using static string literals here instead of random number generation since it is not making a decision on dynamic data.

Reply:
I tend to lean the other way. If I were to pass a static value, the test would only argue about this one value and nothing else